docker-compose/otel/docker-compose.yaml

```diff
@@ -0,0 +1,65 @@
+version: '2.4'
+services:
+  otel-collector:
+    container_name: otel-collector
+    image: otel/opentelemetry-collector:0.54.0
+    volumes:
+      - '../otel-collector/config.yaml:/etc/otel-collector/config.yaml'
+    cpus: 0.5
+    mem_limit: '512m'
+    ports:
+      - '0.0.0.0:4317:4317'
+      - '0.0.0.0:55679:55679'
```

Comment:
is it necessary to bind these ports to the host?

I think each service will only send telemetry to `otel-collector:4317` so traffic won't go outside of the `sourcegraph` network

I assume `zpages` is some sort of debugging web interface? I think we can remove the port binding of `56679` but only include it in the `dev/docker-compose.yaml` override.

Reply:
Note this mirrors how Jaeger binds _everything_ to the host: https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-docker/-/blob/docker-compose/docker-compose.yaml?L528-541

Reply:
That said, `zpages` is not really that useful either, so I will just remove these

Reply:
Removed in 0bebd7ebdd14b5c8e615c66371288f96f9818276, and tested these still work